### PR TITLE
Cypress tests for Events in site-features page

### PR DESCRIPTION
**Pull request by sangeetha-nayak**
GA events test scripts for site-features step from both general onboarding and ecommerce onboarding
Screenshot (447)
Screenshot (448)

### CODE REVIEW

File: tests/cypress/integration/2-general-onboarding-flow/site-features.cy.js [hunk 1]
```diff
@@ -36,19 +37,18 @@ describe( 'Site Features', function () {
...
+			CheckIllustrationPanel();
+			CheckInfoPanel();
+			CheckHelpPanelLinks();
+		} );
+	} else {
 		it( 'Check to make sure Sidebar opens', () => {
 			BasicSidebarCheck();
 		} );
-	};
+	}
```
Comment (avneet-raj) [inline tests/cypress/integration/2-general-onboarding-flow/site-features.cy.js:51]: Why have we removed ';' at the end?

File: tests/cypress/integration/wp-module-support/EventsApi.cy.js [hunk 2]
```diff
@@ -18,7 +18,11 @@ export const APIList = {
 	basic_info:
 		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=wp-setup&_locale=user',
 	basic_info_ecomm:
-		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=ecommerce&_locale=user'
+		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=ecommerce&_locale=user',
```
Comment (avneet-raj) [inline tests/cypress/integration/wp-module-support/EventsApi.cy.js:21]: we are using the same API for general and ecomm flow, so I think it's better to have only 2 APIs, one for general and one for ecomm.
also, please update where the APIs have been used.

File: tests/cypress/integration/2-general-onboarding-flow/site-features.cy.js (hunk 1 above)
Comment (sangeetha-nayak) [reply tests/cypress/integration/2-general-onboarding-flow/site-features.cy.js:51]: Its due to linting. I used command provided earlier.
`npx wp-scripts lint-js <file-path>  --no-ignore --fix`

File: tests/cypress/integration/wp-module-support/EventsApi.cy.js (hunk 2 above)
Comment (sangeetha-nayak) [reply tests/cypress/integration/wp-module-support/EventsApi.cy.js]: Done

**Review by avneet-raj — APPROVED** [review]

**Pull request merged**